drivers/hmis/lib/form_data/assessments/base_assessment.json

```diff
@@ -1359,234 +1359,303 @@
-          "text": "Is this a disabling condition?",
-          "field_name": "mentalHealthDisorderIndefiniteAndImpairs",
-          "pick_list_reference": "NoYesReasonsForMissingData",
-          "disabled_display": "PROTECTED",
-          "enable_behavior": "ALL",
-          "enable_when": [
-            {
-              "question": "4.09.2",
-              "operator": "EQUAL",
-              "answer_code": "YES"
-            }
-          ]
-        },
-
-        {
-          "type": "CHOICE",
-          "link_id": "4.10.2",
-          "text": "Substance Use Disorder",
-          "field_name": "substanceUseDisorder",
-          "pick_list_reference": "DisabilityResponse"
-        },
-        {
-          "type": "CHOICE",
-          "link_id": "4.10.A",
-          "text": "Is this a disabling condition?",
-          "field_name": "substanceUseDisorderIndefiniteAndImpairs",
-          "pick_list_reference": "NoYesReasonsForMissingData",
-          "disabled_display": "PROTECTED",
-          "enable_behavior": "ANY",
-          "enable_when": [
-            {
-              "question": "4.10.2",
-              "operator": "EQUAL",
-              "answer_code": "ALCOHOL_USE_DISORDER"
-            },
-            {
-              "question": "4.10.2",
-              "operator": "EQUAL",
-              "answer_code": "DRUG_USE_DISORDER"
-            },
-            {
-              "question": "4.10.2",
-              "operator": "EQUAL",
-              "answer_code": "BOTH_ALCOHOL_AND_DRUG_USE_DISORDERS"
-            }
-          ]
-        },
-        {
-          "type": "DISPLAY",
-          "link_id": "3.08-info",
-          "text": "<b>OVERALL DISABLING CONDITION</b>: This field is auto-filled based on the above disabilities. The user may override a system-generated NO with a YES.",
-          "_ignore": "HUD defines a Disabling Condition as:<ul><li>A physical, mental, or emotional impairment, including an impairment caused by alcohol or drug abuse, post-traumatic stress disorder, or brain injury that:</li><ul><li>Is expected to be long-continuing or of indefinite duration;</li><li>Substantially impedes the individual's ability to live independently; and</li><li>Could be improved by the provision of more suitable housing conditions.</li></ul></li><li>A developmental disability, as defined in section 102 of the Developmental Disabilities Assistance and Bill of Rights Act of 2000 (42 U.S.C. 15002); or</li><li>The disease of acquired immunodeficiency syndrome (AIDS) or any condition arising from the etiologic agency for acquired immunodeficiency syndrome (HIV).</li></ul>"
-        },
-        {
-          "type": "CHOICE",
-          "link_id": "3.08",
-          "text": "Disabling Condition",
-          "field_name": "disablingCondition",
-          "pick_list_reference": "NoYesReasonsForMissingData",
-          "autofill_values": [
+          "type": "GROUP",
+          "link_id": "disability-table",
+          "component": "INPUT_TABLE",
```

Comment:
Here's what the input table currently looks like with https://github.com/greenriver/hmis-frontend/pull/118

<img width="908" alt="Screen Shot 2023-01-04 at 4 40 22 PM" src="https://user-images.githubusercontent.com/5333982/210655860-e59b5977-6465-4120-a332-a99f0d9a031b.png">